technique/reflectometry/Runscript_day2.py

```diff
@@ -0,0 +1,228 @@
+### This script was generated on 15/02/2022, at 16:55:10
+### with ScriptMaker (c) Maximilian Skoda 2020 
+### Enjoy and use at your own risk. 
+
+import logging
+import sys
+import os
+
+
+try:
+    # pylint: disable=import-error
+    from genie_python import genie as g
+except ImportError:
+    os.environ['instrument'] = "POLREF"
+    from mocks import g
+
+from technique.reflectometry import *  # __all__ defined in __init__.py
+
+# Remove all handlers associated with the root logger object.
+for handler in logging.root.handlers[:]:
+    logging.root.removeHandler(handler)
+
+# Set up new logger
+logging.basicConfig(
+    handlers=[
+        logging.StreamHandler(sys.stdout),
+        logging.FileHandler("example.log", mode="w"),
+    ],
+    level=logging.DEBUG,
```

Comment:
do you definitely need logging.DEBUG here and not logging.INFO? I think this will log absolutely everything going to stdout even from outside of this repo.

Reply:
My intention was a different one - I assigned logging level numbers to our run actions (see script_actions.py) with the intent to get a concise log of what was run. There's probably a better way. I think we can leave it in for now, unless you think it will cause an issue.

Reply:
On SURF it doesn't seem to log anyway, so need looking into

Reply:
> My intention was a different one - I assigned logging level numbers to our run actions (see script_actions.py) with the intent to get a concise log of what was run. There's probably a better way. I think we can leave it in for now, unless you think it will cause an issue.

probably not othre than spamming the console, but i'm only guessing!

Reply:
Hi Jack,

A couple of notes here - some advice would be much appreciated:
- the idea was to define specific log levels with a certain name in order to generate a log file containing something like:
2024-07-29 11:57:44 | RUN | 1 | ** Si1-C19, th=0.7, uamps=10 **
2024-07-29 11:57:44 | RUN | 2 | ** Si1-C19, th=2.3, uamps=30 **
2024-07-29 11:57:44 | CONTRASTCHANGE | Valve 1, concentrations [38, 62, 0, 0], flow 1.0,  volume 15, time None, and wait 
2024-07-29 11:57:44 | RUN | 3 | ** Si1-C19, th=0.7, sec=300 **
...
where the number after RUN would be the actual run number. I think this could be very useful.
- However on SURF it doesn't seem to write anything into that file (even though it does when I'm testing on my machine)
- it doesn't seem to produce any other log message in the console because none of the rest of the code is using logging as far as I can tell.

Ay suggestion on how to make this work?